jsx/FilterableDataTable.js

```diff
@@ -41,36 +53,112 @@ class FilterableDataTable extends Component {
     this.updateFilter({});
   }
 
+  /**
+   * Updates page state
+   *
+   * @param {int} number of page
+   */
+  updatePageNumber(number) {
+    const page = this.state.page;
```

Comment:
what happens if number > the maximum number of pages?

Reply:
I believe given the current GUI, this can't happen. Since updatePageNumber is only triggered when clicking on a `PaginationLink` and `Pagination Link` will only produce the proper amounts of links necessary for the given amount of rows and rows/page.